CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Browserify has been replaced with Webpack which also now builds the SCSS files.
 - Update the canvas to use the existing svg sprites.
 - Removed unused svg assets from the build/dist.
+- GUI is now being served by Flask.
```

Comment:
I'm not sure whether we should keep this CHANGELOG.md. Besically, it implies that this is a versioned package, which I know it used to be.

However, going forward, if it's to follow the pattern of our other websites (which I assume it will), we'll remove any `setup.py` that would imply this might be a package, and slim down `package.json` so it doesn't have any version info, to indicate that this is now purely a live application project without versioning.

In that world, we will be doing releases as often as there are changes, which might be multiple times a day. We might even move to a more automated release process where every merge to master gets pushed straight to at least staging automatically. In that world I don't think it's feasible to keep a changelog. The best way to see changes in that world is probably just to look at the list of closed pull requests.

Thoughts?

Reply:
That may be the case for the JAAS release but the Juju core version requires versioned releases and extensive QA on each one.

Reply:
Okay yeah let's leave this here for now, and we should discuss down the line how we can best facilitate a decent speed of development while also satisfying what Juju needs.